src/TestingUtils/Microsoft.AspNetCore.Testing/ref/Microsoft.AspNetCore.Testing.net46.cs

```diff
@@ -295,6 +295,7 @@ public sealed partial class TestFileOutputContext
         public string TestName { [System.Runtime.CompilerServices.CompilerGeneratedAttribute] get { throw null; } }
         public static string GetAssemblyBaseDirectory(System.Reflection.Assembly assembly, string baseDirectory = null) { throw null; }
         public static string GetOutputDirectory(System.Reflection.Assembly assembly) { throw null; }
+        public static bool GetPreserveExistingLogsInOutput(System.Reflection.Assembly assembly) { throw null; }
```

Comment:
You won't need to do this if the ref/ assembly removal happens first. But, I guess that has to wait for branches to open ☹️

Reply:
Yup